actions/campaigns.go

```diff
@@ -381,3 +381,61 @@ func GetCampaignOpens(c *gin.Context) {
 		"message": "Id must be an integer",
 	})
 }
+
+func GetCampaignStats(c *gin.Context) {
+	if id, err := strconv.ParseInt(c.Param("id"), 10, 64); err == nil {
```

Comment:
@Dzalevski switch the if statement and return early, we should not put the whole code inside an if statement.

Reply:
@FilipNikolovski done. added tests.

Reply:
@FilipNikolovski Yes,action is returning 0 in case not found but if the err is smth else we should reconsider to return 0 not an error?